ui/app/src/components/PathNavigatorTree/PathNavigatorTreeItem.tsx

```diff
@@ -354,22 +354,26 @@ export function PathNavigatorTreeItem(props: PathNavigatorTreeItemProps) {
                 </IconButton>
               </Tooltip>
             )}
-            {over && (showFilter || Boolean(item.childrenCount)) && (
-              <Tooltip title={<FormattedMessage id="words.filter" defaultMessage="Filter" />}>
-                <IconButton
-                  size="small"
-                  className={classes.iconButton}
-                  onClick={(e) => {
-                    e.preventDefault();
-                    e.stopPropagation();
-                    onClearKeywords();
-                    onFilterButtonClick();
-                  }}
-                >
-                  <SearchRoundedIcon color={showFilter ? 'primary' : 'action'} />
-                </IconButton>
-              </Tooltip>
-            )}
+            {showFilter ||
+              (Boolean(item.childrenCount) && (
```

Comment:
Don't think it matters in this particular case, but gotta be careful how prettier groups conditions if you don't group them yourself.
```suggestion
            {(showFilter || Boolean(item.childrenCount) && (
```

Reply:
Prettier still re-formats to the previous one after setting it as in the suggestion